src/Quasar/Mount/View.purs

```diff
@@ -39,13 +40,13 @@ type Config =
 
 toJSON ∷ Config → Json
 toJSON config =
-  let uri = URI.printAbsoluteURI (toURI config)
+  let (uri ∷ String) = AbsoluteURI.print (toURI config)
```

Comment:
Is the annotation necessary here? I thought `AbsoluteURI.print` was monomorphic.